src/hooks/screen/useChatScreenHook.ts

```diff
@@ -143,6 +144,19 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           'pending'
         );
         currentChatSchema.save(localDb);
+        if (selectedChannel) {
+          const channelList: ChannelList | null = await ChannelList.getSchemaById(
+            localDb,
+            selectedChannel?.id
+          );
+          if (channelList) {
+            channelList.description = message;
+            channelList.lastUpdatedBy = userId;
+            channelList.lastUpdatedAt = new Date().toISOString();
+            channelList.save(localDb);
+          }
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code is responsible for updating the channel's description, last updated by, and last updated at fields in the database. However, there is no error handling if the `getSchemaById` function fails or returns null. Consider adding error handling to ensure the application doesn't crash in case of an error.

```diff
  if (selectedChannel) {
    const channelList: ChannelList | null = await ChannelList.getSchemaById(
      localDb,
      selectedChannel?.id
    );
+   if (!channelList) {
+     throw new Error(`Channel with id ${selectedChannel?.id} not found`);
+   }
    channelList.description = message;
    channelList.lastUpdatedBy = userId;
    channelList.lastUpdatedAt = new Date().toISOString();
    channelList.save(localDb);
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->